src/ci/docker/host-aarch64/dist-arm-linux/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN sh /scripts/rustbuild-setup.sh
 WORKDIR /tmp
 
 COPY scripts/crosstool-ng-build.sh /scripts/
-COPY host-x86_64/dist-arm-linux/arm-linux-gnueabi.defconfig /tmp/crosstool.defconfig
+COPY host-aarch64/dist-arm-linux/arm-linux-gnueabi.defconfig /tmp/crosstool.defconfig
```

Comment:
So it sounds like this is still a normal cross-compilation job, it now just happens to use an aarch64 host in CI. Can we drop this host-aarch64/host-x86_64 distinction for jobs where it is not relevant? Otherwise it will not be possible to test this image locally (on x86_64) anymore.

Reply:
Basically, as far as I can tell, there is no relation at all here between the fact that this is dist-arm-linux and we're running on ARM. You could do exactly the same move for all/most other crosstool-ng based jobs, if the ARM runners are cheaper for oyu.

Reply:
Good point. But based on your comment from the other PR, moving to a native build would raise the minimum glibc version, right? Since this already used crosstool before, we could make use of it to keep glibc lower.